cms/static/sass/_base.scss

```diff
@@ -385,9 +386,9 @@ dl {
   }
 
   // page metadata/action bar
-  .metadata {
+  // .metadata {
```

Comment:
Can't just delete this?

Reply:
We could, but there are a good number of these throughout and I didn't know if they were there for clarity during theming or something

Reply:
Ah, yeah, I just asked the same thing :)

With the same guess, that _maybe_ they're meant as placeholders for theming?
Still seems odd...